daemons/gptp/common/ieee1588port.cpp

```diff
@@ -572,11 +582,9 @@ void IEEE1588Port::processEvent(Event e)
 			Event e4 = NULL_EVENT;
 
 			// TODO:Start PDelay only if the link is up.
-			if (!clock->automotiveProfileEnabled() || !clock->forceAsCapable()) {
-				if (port_state != PTP_SLAVE && port_state != PTP_MASTER) {
+			if (!clock->automotiveProfileEnabled() || !clock->forceAsCapableEnabled()) {
 					GPTP_LOG_STATUS("Starting PDelay");
 					startPDelay();
-				}
```

Comment:
I'm not sure why this port_state check was in here in the first place. What you have looks correct but it might be worth double checking the standard profile's behavior to make sure we aren't breaking anything.